tools/src/py-mkopensource/main.go

```diff
@@ -46,7 +46,7 @@ func parseLicenses(name, version, license string) map[License]struct{} {
 		{"pathspec", "0.12.1", ""}:                        {MPL2},
 		{"ptyprocess", "0.7.0", "UNKNOWN"}:                {ISC},
 		{"python-json-logger", "2.0.7", "BSD"}:            {BSD2},
-		{"ruff", "0.12.7", ""}:                            {MIT},
+		{"ruff", "0.12.8", ""}:                            {MIT},
```

Comment:
This is sneaky hiding in the `make generate` commit, and I am pretty confident this change didn't come from `make generate` :)

Reply:
This is actually a slightly different issue than you called out: `ruff` is a development dependency, not a runtime dependency, and what _really_ needs to happen here is that we need to cleanly separate those in the post-`uv` world. 😐 Soon!